app/assets/stylesheets/content/_badges.md

```diff
@@ -0,0 +1,13 @@
+# Badges
```

Comment:
In the context of flat buttons it makes atleast more sense, that this thing looks so sticker-ish.

I think badge could be okay...